app/sagas/quickActions.ts

```
@@ -0,0 +1,108 @@
+
+	switch (quickAction) {
+		case 'add-server': {
+			const server = UserPreferences.getString(CURRENT_SERVER);
+
+			yield put(appStart({ root: RootEnum.ROOT_OUTSIDE }));
+			yield put(serverInitAdd(server || ''));
+			break;
+		}
+		case 'search':
+			yield waitForAppReady();
+			const currentRoute = Navigation.getCurrentRoute();
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)